.prettierrc

```diff
@@ -0,0 +1,9 @@
+{
+  "semi": false,
+  "singleQuote": true,
+  "bracketSpacing": true,
+  "trailingComma": "es5",
```

Comment:
I don't like trailing commas, but they reduce merge-diffs 🤔
**You** should decide

Reply:
Yeah, I would prefer to keep this.

Reply:
ok